{"name": "Изменение данных неавторизованным пользователем", "status": "passed", "description": "Тест проверяет появление ошибки, при попыткиизменить данные неавторизованным пользователем", "parameters": [{"name": "data_user", "value": "'Pamela Williams'"}], "start": 1724065603577, "stop": 1724065603745, "uuid": "e433ebd9-777b-4a18-ac72-0c1abba9445d", "historyId": "7110ac450caca154e5cfa5e34cd8b736", "testCaseId": "75e3752286952ce47f86b20de866b56d", "fullName": "tests.test_update_user_data.TestUpdateUserData#test_update_user_data_not_authorized", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_user_data"}, {"name": "subSuite", "value": "TestUpdateUserData"}, {"name": "host", "value": "DESKTOP-L3DLL49"}, {"name": "thread", "value": "13308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_user_data"}]}